docs/components/receive.md

```diff
@@ -77,6 +77,25 @@ The example content of `hashring.json`:
 
 With such configuration any receive listens for remote write on `<ip>10908/api/v1/receive` and will forward to correct one in hashring if needed for tenancy and replication.
 
+## Limiting
+
+### Request limits (work in progress)
+
+Thanos Receive supports setting limits on the incoming remote write request sizes. These limits should help you to prevent a single tenant from being able to send big requests and possibly crash the Receive.
+
+These limits are applied per request and can be configured with the following command line arguments:
+
+- `--receive.write-request-limits.max-size-bytes`: the maximum body size.
+- `--receive.write-request-limits.max-concurrency`: the maximum amount of remote write requests that will be concurrently processed while others wait.
```

Comment:
I'm assuming the max concurrency limit won't result in `413` response, maybe it would be worth to put this into a separate section, since it's not really a size limit but rather a gating mechanism. What do you think?

Reply:
Suuure, it makes sense. I'll move it.